README.md

```diff
@@ -1443,7 +1443,7 @@ See https://staticsitegenerators.net and https://www.staticgen.com
 - [Tinyissue](https://github.com/satrun77/tinyissue) - Simple Issue Tracking for Teams. `MIT` `PHP`
 - [todo](https://github.com/prologic/todo) - Simple todo list manager. ([Demo](https://todo.mills.io)) `MIT` `Go`
 - [todoMini](https://www.todomini.app/) - Mobile friendly zero-feature TODO list web app. Unix philosophy. ([Demo](https://appmini.github.io/todoMini/?demo), [Source Code](https://github.com/appMini/todoMini)) `GPL-3.0` `PHP/Java`
-- [Tracks](http://www.getontracks.org/) - Web-based application to help you implement David Allen’s [Getting Things Done™](https://en.wikipedia.org/wiki/Getting_Things_Done) methodology. ([Source Code](https://github.com/TracksApp/tracks)) `GPL-2.0` `Ruby`
+- [Tracks](http://www.getontracks.org/) - Web-based task management application featuring contexts, projects and statistics helping you implement David Allen’s [Getting Things Done™](https://en.wikipedia.org/wiki/Getting_Things_Done) methodology. Features also an extensive API and feeds. Based on Ruby on Rails. ([Demo](https://www.taskitin.fi), [Source Code](https://github.com/TracksApp/tracks), [Clients](https://www.getontracks.org/clients/)) `GPL-2.0` `Ruby/Docker`
```

Comment:
Description should be 250 characters or less.

Reply:
@ZeiP please use a shortened version of the official description at https://www.getontracks.org/:

> web-based application to help you implement David Allen’s Getting Things Done™ methodology